Foundation/NSXMLParser.swift

```diff
@@ -233,9 +233,9 @@ internal func _NSXMLParserStartElementNs(ctx: _CFXMLInterface, localname: Unsafe
     let parser = ctx.parser
     let reportQNameURI = parser.shouldProcessNamespaces
     let reportNamespaces = parser.shouldReportNamespacePrefixes
-    let prefixLen = prefix == nil ? strlen(UnsafePointer<Int8>(prefix)) : 0
+    let prefixLen = prefix == nil ? UInt(strlen(UnsafePointer<Int8>(prefix))) : 0
```

Comment:
`size_t` is supposed to be imported as `Int` in Swift, the importer has a special case for it.  Would you mind checking if there is an issue with that on Linux/armv7?

```Swift
import Glibc
print(size_t.self) // should print 'Int'
```